### PR TITLE
feat(_scripts/deploy.sh/Makefile): automatically deploy to the k8s-claimer-e2e deis cluster

### PR DESCRIPTION
**Pull request by arschles**

### CODE REVIEW

File: _scripts/deploy.sh [hunk 1]
```diff
@@ -10,3 +10,9 @@ docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 DEIS_REGISTRY='' make -C .. docker-build docker-push
 docker login -e="$QUAY_EMAIL" -u="$QUAY_USERNAME" -p="$QUAY_PASSWORD" quay.io
 DEIS_REGISTRY=quay.io/ make -C .. docker-build docker-push
+
+# download deis CLI & deploy to deis
+curl -sSL http://deis.io/deis-cli/install-v2.sh | bash
+mv ./deis /usr/local/bin/deis
+deis login --username=$DEIS_USERNAME --password=$DEIS_PASSWORD
```
Comment (vdice) [inline _scripts/deploy.sh]: Where are `DEIS_USERNAME` and `DEIS_PASSWORD` set? Assume we're expecting them to be set in CI environ -- do we need to set these to some defaults if both/either are empty?

**Comment by bacongobbler** [comment]
should we also add business logic to create the app if it does not exist? This won't work on a bare cluster because the app has not been created yet. This logic also assumes that the app name is either in the git remote or the app name is k8s-claimer (or w/e the subdirectory name is)

**Comment by arschles** [comment]
@bacongobbler :

> should we also add business logic to create the app if it does not exist?

IMO no, because I'm considering a bare cluster at deis.k8s-claimer-e2e.deis.com (which I left out, and that's a bug! Fixing shortly) not suitable for this application to be automatically deployed to.

> This logic also assumes that the app name is either in the git remote or the app name is k8s-claimer (or w/e the subdirectory name is)

Good catch. Fixing shortly

**Comment by arschles** [comment]
@bacongobbler app name & login remote set in https://github.com/deis/k8s-claimer/pull/43/commits/5dd3f80cc37420a6749d9106f911f77e3645593d

File: _scripts/deploy.sh (hunk 1 above)
Comment (arschles) [inline _scripts/deploy.sh]: Yep, we're assuming they'll be set in the CI environment. They're currently set at https://travis-ci.org/deis/k8s-claimer/settings

**Pull request merged**